librz/bin/dwarf.c

```diff
@@ -2034,6 +2037,9 @@ RzBinSection *getsection(RzBinFile *binfile, const char *sn) {
 		return NULL;
 	}
 	rz_list_foreach (o->sections, iter, section) {
+		if (!section->name) {
+			continue;
+		}
```

Comment:
I really don't like this thing everywhere...  I mean, fine for this fix, but IMHO a proper way to deal with this would be to ensure that RzBinSections are always well-formed and have a `->name` always pointing to a string. Like at https://github.com/rizinorg/rizin/blob/dev/librz/bin/bin.c#L1389 and in few other places we should just `strdup("")` IMO.

Reply:
In fact, I think we just should `RZ_IPI RzBinSection *rz_bin_section_new(RZ_NONNULL const char *name)` and `rz_return_val_if_fail(name, NULL)`. And prohibit such calls from every plugin.